docs/setup-ssl.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[ssl-configuration]]
+== SSL/TLS communication with APM Server
+
+If {apm-server-ref-v}/secure-communication-agents.html#ssl-setup[SSL/TLS communication] is enabled on
+the APM Server, use the `https` protocol when configuring <<config-server-urls,`server_urls`>>.
+
+[float]
+[[ssl-server-authentication]]
+=== APM Server certificate authentication
+
+By default, when using HTTPS to communicate with APM Server, the agents will verify the identity
+of the APM Server by authenticating its certificate. It is not recommended to change this default,
+however it is possible through the <<config-verify-server-cert>> configuration.
+
+If the certificate used by the APM Server is self-signed, you would need to add the same certificate
+to the JVM's truststore. You can find which truststore is used by setting `-Djavax.net.debug=all`
+in the command line. Typically, it would be `$JAVA_HOME/jre/lib/security/cacerts`.
+Following is an example of using `keytool` to import a certificate into the truststore:
+`keytool -import -alias apm-server-cert -file /path/to/certificate.crt -keystore /path/to/truststore`
```

Comment:
This is a long command. It probably makes more sense to format this in a code block so that it doesn't get split onto multiple lines
```suggestion
[source,bash]
----
keytool -import -alias apm-server-cert -file /path/to/certificate.crt -keystore /path/to/truststore
----
```